From bf78414abb7014966c4fefd49f6c25502bbc4f9d Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Wed, 4 Nov 2009 14:24:42 +0000
Subject: [PATCH] Add default media application schema

So that there is a default media player selected in the
"Default Applications" capplet.

https://bugzilla.gnome.org/show_bug.cgi?id=435653
---
 schemas/desktop_default_applications.schemas.in |   22 ++++++++++++++++++++++
 1 files changed, 22 insertions(+), 0 deletions(-)

diff --git a/schemas/desktop_default_applications.schemas.in b/schemas/desktop_default_applications.schemas.in
index 515aee0..aa47ba5 100644
--- a/schemas/desktop_default_applications.schemas.in
+++ b/schemas/desktop_default_applications.schemas.in
@@ -24,6 +24,28 @@
       </locale>
     </schema>
     <schema>
+      <key>/schemas/desktop/gnome/applications/media/exec</key>
+      <applyto>/desktop/gnome/applications/media/exec</applyto>
+      <owner>gnome-vfs</owner>
+      <type>string</type>
+      <default>gnome-mplayer</default>
+      <locale name="C">
+        <short>Default media application</short>
+        <long>The default media application to use when media buttons are pressed.</long>
+      </locale>
+    </schema>
+    <schema>
+      <key>/schemas/desktop/gnome/applications/media/exec_arg</key>
+      <applyto>/desktop/gnome/applications/media/exec_arg</applyto>
+      <owner>gnome-vfs</owner>
+      <type>string</type>
+      <default></default>
+      <locale name="C">
+        <short>Exec argument for default media application</short>
+        <long>The exec argument to use for the default media application.</long>
+      </locale>
+    </schema>
+    <schema>
       <key>/schemas/desktop/gnome/applications/component_viewer/exec</key>
       <applyto>/desktop/gnome/applications/component_viewer/exec</applyto>
       <owner>gnome-vfs</owner>
-- 
1.6.5.1

